lib/embulk/input/mixpanel.rb

```diff
@@ -68,7 +65,8 @@ def self.transaction(config, &control)
           incremental_column: incremental_column,
           retry_limit: config.param(:retry_limit, :integer, default: 5),
           latest_fetched_time: latest_fetched_time,
-          incremental: incremental
+          incremental: incremental,
+          slice_range: config.param(:slice_range, :integer, default: 7)
```

Comment:
`default: DEFAULT_FETCH_DAYS` ?

Reply:
@sakama It's just happen to be the same amount, they are different things.

Reply:
OK. I approved. LGTM